ReactWindows/ReactNative.Shared/Animated/NativeAnimatedNodesManager.cs

```diff
@@ -107,6 +108,7 @@ public void CreateAnimatedNode(int tag, JObject config)
 
             _animatedNodes.Add(tag, node);
             _updatedNodes[tag] = node;
```

Comment:
>_updatedNodes [](start = 12, length = 13)

You said your strategy was to call `ActivateCallback` whenever an item was added to `_activeAnimations` or `_updatedNodes` because that's what `HasActiveAnimations` checks for and we call `DeactivateCallback` when `HasActiveAnimations` returns `false.`

There are a number of other places where an item is set on `_updatedNodes`. Do we need to call `ActivateCallback` in any of those other places?

Reply:
I added just to methods that looked like were adding items to those collections. The others seem to just update the values for existing keys
Anyway, there's something I don't want about the solution: these methods are also called through the orchestrator, so it's not the right place to "initiate" subscriptions.
Coming up with something better.

---
In reply to: [164948241](https://github.com/Microsoft/react-native-windows/pull/1608#discussion_r164948241) [](ancestors = 164948241)